sprints/sustainability/models.py

```diff
@@ -309,54 +366,3 @@ def get_accounts_remaining_time(self) -> Dict[str, float]:
                 accounts[account.name] += account.ytd_goal - account.ytd_overall
 
         return accounts
-
-
-class Budget(models.Model):
-    """
-    Stores monthly budgets for accounts.
-    If the budget for the specific month is not present, the last existing one is used.
-    """
-    name = models.CharField(max_length=255, help_text="Account's name.")
-    date = models.DateField(
-        help_text="Year and month of the budget. If not specified, the last month's budget is applied."
-    )
-    hours = models.IntegerField(help_text="Number of available hours for this month.")
-
-    def __str__(self):
-        return f"{self.name}: {self.date.strftime('%Y-%m')}"
-
-
-class Account(models.Model):
-    """
-    Stores email addresses for sending notifications about problems with the budgets of the account.
-    """
-    name = models.CharField(max_length=255, help_text="Account's name.")
-    alert_emails = ArrayField(
```

Comment:
@Agrendalath How did we replace this? Are some emails still being notified?

Reply:
I just had to move `Budget` model to the top of this file, because `typing` was not detecting it from the bottom. I moved `Account` and `Cell` as well to have the consistency of class definitions.